Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -295,6 +295,35 @@ extension BrowserViewController: URLBarDelegate {
         }
     }
 
+    func urlBarDidLongPressLocation(urlBar: URLBarView) {
+        let longPressActionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .ActionSheet)
+
+        let pasteboardContents = UIPasteboard.generalPasteboard().string
+
+        // Check if anything is on the pasteboard
+        if ((pasteboardContents) != nil) {
```

Comment:
Nit: we generally try to avoid using parentheses when not necessary, so I'd remove both sets of them in this case.